src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -143,8 +143,8 @@ private FolderComposite FindProjectFilesUsingBuildalyzer(ProjectAnalyzerResult a
             var rootFolderComposite = new FolderComposite()
             {
                 Name = string.Empty,
-                FullPath = projectRoot,
-                RelativePath = string.Empty,
+                FullPath = projectUnderTestDir,
```

Comment:
We need a top level 'container' FolderComposite in case there are multiple projects, so we can't just take the projectUnderTestDir as the root folder. It needs to be one folder level higher.

Reply:
I did some changes, I think it does more sense now. 

But then, I don't see the meaning in calling the property `ProjectRoot` as it's a bit misleading as paths are relative to the parent that holds all the projects in the case you have many.

Reply:
You are correct, it's probably a bad name. How about solutionRoot?

Reply:
Done!

Reply:
Hi sorry, I thought you were talking about the naming of the projectRoot variable in InputFileResolver.cs.


In the context of the mutation report schema I don't think it makes sense to call it SolutionRoot instead of ProjectRoot as dotnet has solutions and solution files but other languages and frameworks do not have this concept. The mutation report schema is generic, and in other places where we deal with the schema we call the 'combination' of 'modules' a project. 

Perhaps we should have the discussion about what we call it in mutation-testing-metrics instead of here since we are not the only consumers of the json schema.